wsd/DocumentBroker.hpp

```diff
@@ -775,16 +769,10 @@ class DocumentBroker : public std::enable_shared_from_this<DocumentBroker>
         }
 
         /// Set the last modified time of the document.
-        void setLastModifiedTime(std::chrono::system_clock::time_point time)
-        {
-            _request.setModifiedTime(time);
-        }
+        void setLastModifiedTime(std::chrono::system_clock::time_point time) { _modifiedTime = time; }
```

Comment:
Now we have a setLastModifiedTime here that works with the time_point, and the other that works with a string.  I wonder - could or should we rename one of these, for better understanding?

But I guess that's not blocking the integration - so feel free to update in a follow-up PR if you agree with the above :-)

Reply:
Makes sense. Thanks for the review!

Reply:
Done, waiting for the build.